recipes/pyxid/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyxid" %}
+{% set version = "1.0" %}
+{% set sha256 = "e689bce2ed8ed4bff7174a3a79f8b639ebf6356807aec7e9da481217313214ff" %}
+
+package:
+  name: pyxid
+  version: "1.0"
+
+source:
+  fn: pyxid-{{ version }}.tar.gz
+  url: https://github.com/cedrus-opensource/pyxid/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyserial >=2.5
+
+
+  run:
+    - python
+    - pyserial >=2.5
+
+test:
+  # Python imports
+  imports:
+    - pyxid
+
+about:
+  home: http://www.github.com/cedrus-opensource/pyxid/
+  license: BSD
+  summary: 'Pure python library for communicating with Cedrus XID and StimTracker devices.'
+  license_family: BSD
+  author: "Grant Limberg"
+  author_email: "developers@cedrus.com"
+  maintainer: "Cedrus Corporation"
+  maintainer_email: "opensource@cedrus.com"
```

Comment:
Sorry, I just grabbed these all from the ``setup.py``; I've taken them back out. Extra keys in the yaml don't break ``conda-build``, but I think it's cleaner not to have them.